### PR TITLE
BakeFirstElements: moved selectors from recipe to kitchen

### PR DESCRIPTION
**Pull request by KarinaMendez2**

### CODE REVIEW

File: lib/kitchen/directions/bake_first_elements.rb [hunk 1]
```diff
@@ -3,7 +3,13 @@
 module Kitchen
   module Directions
     module BakeFirstElements
-      def self.v1(within:, selectors:)
+      def self.v1(within:)
+        selectors = [
+          '.os-problem-container > .os-table',
```
Comment (jpslav) [inline lib/kitchen/directions/bake_first_elements.rb:8]: please add the tag names here as you can per https://openstax.slack.com/archives/C022NETP39N/p1621863752137000

**Review by jpslav — CHANGES_REQUESTED** [review]
Some test coverage is needed too.

**Review by kswanson33 — CHANGES_REQUESTED** [review]

File: spec/directions/bake_first_elements_spec.rb [hunk 2]
```diff
@@ -175,7 +172,7 @@
                   <div data-type="problem"></div>
                   <div data-type="solution">
                     <div class="os-solution-container has-first-element">
-                      <div class="first-element" data-type="media">Media</div>
+                      <span class="first-element" data-type="media">Media</div>
```
Comment (kswanson33) [inline spec/directions/bake_first_elements_spec.rb:175]: and this one

**Review by kswanson33 — APPROVED** [review]
❤️

**Review by jpslav — APPROVED** [review]

**Pull request merged**